yellowbrick/cluster/silhouette.py

```diff
@@ -285,26 +285,31 @@ def silhouette_visualizer(
         or ``MiniBatchKMeans``). If the estimator is not fitted, it is fit when
         the visualizer is fitted, unless otherwise specified by ``is_fitted``.
 
-    ax : matplotlib Axes, default: None
-        The axes to plot the figure on. If None is passed in the current axes
-        will be used (or generated if required).
-
     X : array-like of shape (n, m)
         A matrix or data frame with n instances and m features
 
     y : array-like of shape (n,), optional
         A vector or series representing the target for each instance
 
+    ax : matplotlib Axes, default: None
+        The axis to plot the figure on. If None is passed in the current axes
+        will be used (or generated if required).
+
```

Comment:
Thank you for re-arranging the docstring params to match the function signature!